src-ts/utils/account/styles.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../lib/styles';
+
+.page-container {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    margin: 0 $pad-xxxxl;
+
+    h1 {
+        color: #2A2A2A;
+        border-bottom: 1px solid $black-10;
+    }
+
+    .page-content {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        margin: $pad-xxxxl -#{$pad-md} 0;
+
+        @include ltemd {
+            flex-direction: column;
+        }
+
+        .card {
```

Comment:
i assume you switched to the global styles so that you could use this .card style.

rather than explicitly styling the card, can we use display: grid to size the cards? 

```
.page-content {
    display: grid;
    grid-template-columns: repeat(3, 1fr);
    column-gap: 12px;

     @include ltemd {
         grid-template-column; 1fr;
     }
}
```